apps/scoutgame/app/api/builders/search/route.ts

```diff
@@ -1,20 +1 @@
-import { log } from '@charmverse/core/log';
-import { searchBuilders } from '@packages/scoutgame/builders/searchBuilders';
-
-export async function GET(request: Request) {
-  const { searchParams } = new URL(request.url);
-  const search = searchParams.get('search');
-  if (typeof search !== 'string') {
-    return new Response('path is required', { status: 400 });
-  }
-  try {
-    const result = await searchBuilders({
-      search
-    });
-
-    return Response.json(result);
-  } catch (error) {
-    log.error('Error requesting user from farcaster', { error, search });
-    return new Response(`Unknown error: ${(error as Error).message}`, { status: 500 });
-  }
-}
+export * from '@packages/scoutgame-router/api/builders/search/route';
```

Comment:
a new pattern to reuse api endpoints and other app endpoints. unfortunately, symlinks are not respected by next.js when i test in dev